src/winter_onboarding_2021/fleet_management_service/handlers/invites.clj

```diff
@@ -0,0 +1,49 @@
+(ns winter-onboarding-2021.fleet-management-service.handlers.invites
```

Comment:
the file file should be named as `invite` in order to be consistent with other namespaces. wdyt?